core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4854,6 +4868,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String UNDERFS_KODO_CONNECT_TIMEOUT =
         "alluxio.underfs.kodo.connect.timeout";
     public static final String UNDERFS_KODO_REQUESTS_MAX = "alluxio.underfs.kodo.requests.max";
+    public static final String B2_ACCESS_KEY = "b2AccessKeyId";
+    public static final String B2_SECRET_KEY = "b2SecretKey";
```

Comment:
Can these follow the conventions of other property keys by something of the form `alluxio.underfs.*` with words separated by `.`? e.g. I'm thinking of `alluxio.underfs.b2.access.key.id`

Reply:
done